4.0.0/stability

rart opened the pull request:
- Add GraphQL icon
- Item actions clean up
- Fix found runtime exception due to unhandled optional prop
- Fix found runtime exception due to path sometimes not containing "/index.xml"
- Avoid runtime exceptions in item inspection functions supplied with null item
- Fix Graphql icon copyright
- Falling back to default framework input styles
- Adjusting customised input styles
- Add missing translations

rart committed:
4.0.0 Stability
- GraphQL icon
- Item actions clean up
- Fix found runtime exception due to unhandled optional prop
- Fix found runtime exception due to path sometimes not containing "/index.xml"
- Avoid runtime exceptions in item inspection functions supplied with null item

rart committed:
Merge branch 'develop' of https://github.com/craftercms/studio-ui into 4.0.0/develop

sumerjabri commented on ui/app/src/components/Icons/GraphQL.tsx:
```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright (C) 2007-2020 Crafter Software Corporation. All Rights Reserved.
```
Wrong copyright notice date. Please copy/paste latest from Slack.

rart replied:
Done.

sumerjabri commented on ui/app/src/components/Icons/GraphQL.tsx:
```diff
@@ -0,0 +1,31 @@
+    <path d="M3.8244 8.84449L3.35811 8.03691L12.3586 2.8403L12.8249 3.64794L3.8244 8.84449Z" />
+    <path d="M20.0095 18.3826L11.0123 2.79949L11.82 2.3332L20.8171 17.9163L20.0095 18.3826ZM3.7136 7.4184H4.64618V17.8116H3.7136V7.4184ZM19.354 7.4184H20.2866V17.8116H19.354V7.4184Z" />
+    <path d="M12.1975 21.9972L11.7902 21.2917L19.6183 16.772L20.0256 17.4775L12.1975 21.9972Z" />
+    <path d="M21.5222 18.1152C20.9829 19.0534 19.7807 19.3736 18.8425 18.8343C17.9043 18.2949 17.5841 17.0927 18.1234 16.1545C18.6627 15.2163 19.8649 14.8961 20.8031 15.4354C21.747 15.9803 22.0672 17.177 21.5222 18.1152ZM5.87062 9.07588C5.3313 10.0141 4.12906 10.3343 3.19086 9.79497C2.25266 9.25565 1.93244 8.05341 2.47176 7.11521C3.01108 6.17701 4.21333 5.85679 5.15152 6.39611C6.08972 6.94105 6.4 ...
```
Can this line be broken up?

rart replied:
I'm not sure. Not worth the effort anyway as these are sources generated by the design tool.

rart committed:
4.0.0 Stability
- Fix Graphql icon copyright
- Falling back to default framework input styles
- Adjusting customised input styles
- Add missing translations

rart committed:
4.0.0 Stability
- Publishing dialogs form style fixes

rart committed:
4.0.0 Stability
- LoginForm style fixes
